core/trino-main/src/main/java/io/trino/operator/TableCoordinatorOnlyExecuteOperator.java

```diff
@@ -0,0 +1,139 @@
+        private final TableCoordinatorExecuteTarget executeTarget;
+        private boolean closed;
+
+        public TableCoordinatorOnlyExecuteOperatorFactory(
+                int operatorId,
+                PlanNodeId planNodeId,
+                Metadata metadata,
+                Session session,
+                TableCoordinatorExecuteTarget executeTarget)
+        {
+            this.operatorId = operatorId;
+            this.planNodeId = requireNonNull(planNodeId, "planNodeId is null");
+            this.metadata = requireNonNull(metadata, "planNodeId is null");
+            this.session = requireNonNull(session, "planNodeId is null");
+            this.executeTarget = requireNonNull(executeTarget, "planNodeId is null");
+        }
+
+        @Override
+        public Operator createOperator(DriverContext driverContext)
+        {
+            checkState(!closed, "Factory is already closed");
+            OperatorContext context = driverContext.addOperatorContext(operatorId, planNodeId, TableCoordinatorOnlyExecuteOperator.class.getSimpleName());
+            return new TableCoordinatorOnlyExecuteOperator(context, metadata, session, executeTarget);
+        }
+
+        @Override
+        public void noMoreOperators()
+        {
+            closed = true;
+        }
+
+        @Override
+        public OperatorFactory duplicate()
+        {
+            return new TableCoordinatorOnlyExecuteOperatorFactory(operatorId, planNodeId, metadata, session, executeTarget);
+        }
+    }
+
+    private final OperatorContext operatorContext;
+    private final Metadata metadata;
+    private final Session session;
+    private final TableCoordinatorExecuteTarget executeTarget;
+
+    private boolean finished;
+
+    public TableCoordinatorOnlyExecuteOperator(OperatorContext operatorContext, Metadata metadata, Session session, TableCoordinatorExecuteTarget executeTarget)
+    {
+        this.operatorContext = requireNonNull(operatorContext, "operatorContext is null");
+        this.metadata = requireNonNull(metadata, "metadata is null");
+        this.session = requireNonNull(session, "session is null");
+        this.executeTarget = requireNonNull(executeTarget, "executeTarget is null");
+    }
+
+    @Override
+    public OperatorContext getOperatorContext()
+    {
+        return operatorContext;
+    }
+
+    @Override
+    public boolean needsInput()
+    {
+        return false;
+    }
+
+    @Override
+    public void addInput(Page page)
+    {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public Page getOutput()
+    {
+        if (finished) {
+            return null;
+        }
+        finished = true;
+
+        metadata.executeTableExecute(session, executeTarget.getExecuteHandle());
+        PageBuilder page = new PageBuilder(1, TYPES);
+        BlockBuilder rowsBuilder = page.getBlockBuilder(0);
+        page.declarePosition();
+        VARCHAR.writeString(rowsBuilder, "SUCCESS");
```

Comment:
It feels to me output schema of all `ALTER TABLE ... EXECUTE` statements should be the same. So I would make it single BIGINT column and return no rows.